hybrid/compartment_examples/inter_comp_call/secure/build/Makefile.inter_comp_call

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2020-2021 The CapableVMs "CHERI Examples" Contributors.
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+# Common Makefile for the shared_object example.
+# This should not be invoked directly.
+
+ifndef BINDIR
+$(error build/Makefile.vars should set BINDIR)
+endif
+
+ifdef RUNDIR
+RUNDIR := $(RUNDIR)/inter_comp_call
+else
+RUNDIR := inter_comp_call
+endif
+
+UPDATE_DDC_EXAMPLE := secure-update_ddc
```

Comment:
Is this a list, or how would it look like adding more examples to this variable? I would perhaps say a few words what this will do, even if it is mentioned a bit later on.

Reply:
It's meant to be just one (if we want this to be a list instead, I'll need to rework the part where `UPDATE_DDC_EXAMPLE` is used).

Reply:
I think if it's dynamic it should either be a list, so it can be updated at will, or it should be static, so we hardcode saying which examples we want to leak. The cleanest and most future proof would be if we can somehow make it a list, but I don't know how difficult that might be. An alternative would be adding a new `make` command, like `make run-leak-{example}` or something like that.

Reply:
I think this can easily be rewritten to support lists. I'll give it a try and push a commit for you to review.

Reply:
Think it looks cleaner as well. Nice.